zulipterminal/config/keys.py

```diff
@@ -464,6 +466,42 @@ def commands_for_random_tips() -> List[KeyBinding]:
     ]
 
 
+def transform_key_binding(keys: List[str]) -> str:
+    display_key = []
+    for key in keys:
+        if key in SPECIAL_KEYS_MAPPINGS:
+            display_key.append(SPECIAL_KEYS_MAPPINGS[key])
+        else:
+            display_key.append(key)
+    return " ".join(display_key)
+
+
+def display_keys_for_command(
+    command: str, help_text: bool = False, key_list: List[str] = []
+) -> List[str]:
+    keys = []
+    urwid_key_list = []
+    if help_text:
+        urwid_key_list = key_list
+    else:
+        try:
+            urwid_key_list = KEY_BINDINGS[command]["keys"]
+        except KeyError as exception:
+            raise InvalidCommand(command) from exception
```

Comment:
The extra arguments were basically to accommodate the behavior in HelpView ([exact line](https://github.com/zulip/zulip-terminal/blob/f750cd15698382a92d73e8ec13efcb757b2f6111/zulipterminal/ui_tools/views.py#L1196)). Here the keys are rendered directly from a list fetched earlier and not the command. The second place in which a list is used is while generating the random help command ([line](https://github.com/zulip/zulip-terminal/blob/f750cd15698382a92d73e8ec13efcb757b2f6111/zulipterminal/ui.py#L99)).

But on revisiting this, I believe we can refactor the code a bit in HelpView (use KEY_BINDINGS.items() instead of values() would be one of the main changes) and eliminate the need for those extra arguments. Or perhaps you have another suggestion for how we should approach this?